### PR TITLE
Rebranded Windows installer

### PR DESCRIPTION
**Pull request by MarcusLlewellyn**
This re-brands the installer created by the PACKAGE target so that it uses Project Athena text and images. It also removes the slideshow and all of the Google Analytics. The scripting code for the slideshow and analytics is still present, just currently commented out.

Currently, this still bundles up and offers to install the High Fidelity system tray launcher.

### CODE REVIEW

File: cmake/templates/NSIS.template.in [hunk 1]
```diff
@@ -161,7 +161,7 @@
 ;General
 
   ; hide install details since we show an image slideshow in their place
-  ShowInstDetails nevershow
+  ShowInstDetails hide
```
Comment (thoys) [inline cmake/templates/NSIS.template.in]: What does this do?

Comment (MarcusLlewellyn) [reply cmake/templates/NSIS.template.in]: This is the part of the installer where the slideshow used to appear. Now, it shows the "Show Details" button common to NSIS installers.

File: cmake/macros/SetPackagingParameters.cmake [hunk 2]
```diff
@@ -192,21 +192,21 @@ macro(SET_PACKAGING_PARAMETERS)
...
-      set(INTERFACE_SHORTCUT_NAME "High Fidelity - ${BUILD_VERSION_NO_SHA}")
+      set(INTERFACE_SHORTCUT_NAME "Project Athena - ${BUILD_VERSION_NO_SHA}")
       set(CONSOLE_SHORTCUT_NAME "Console - ${BUILD_VERSION_NO_SHA}")
       set(SANDBOX_SHORTCUT_NAME "Sandbox - ${BUILD_VERSION_NO_SHA}")
     endif ()
 
     set(INTERFACE_HF_SHORTCUT_NAME "${INTERFACE_SHORTCUT_NAME}")
-    set(CONSOLE_HF_SHORTCUT_NAME "High Fidelity ${CONSOLE_SHORTCUT_NAME}")
-    set(SANDBOX_HF_SHORTCUT_NAME "High Fidelity ${SANDBOX_SHORTCUT_NAME}")
+    set(CONSOLE_HF_SHORTCUT_NAME "Project Athenay ${CONSOLE_SHORTCUT_NAME}")
```
Comment (kasenvr) [inline cmake/macros/SetPackagingParameters.cmake]: Nani?

Comment (MarcusLlewellyn) [reply cmake/macros/SetPackagingParameters.cmake]: Fixed!

**Pull request merged**